RoadTripMoustache/kana_to_kanji
# feat: Reload all static data

## 📖 Description
If the app receive an error 404 from `GET /sync`, it means that the app version is obsolete. So the app must clean all the collections with static data and call the API to all download again.

### ⁉️ Related Issues
closes: #222 

### 🧪 How to test the change?
- Open the API logs
- Open the application
- Close the application
- Update the database to create a new version with a new kanji and delete all the others
- Open the application
- In the API logs, `GET /sync` must return an error 404 and all `GET /...` endpoints must be called
- Check kanjis in the glossary, you must only see the new kanji

### ☑️ Checklist before requesting a review
